### DISCUSSION TITLE
Running optimize without resampling animation?

### DISCUSSION THREAD

**Discussion by ketz** [donmccurdy/glTF-Transform#1583] [Q&A] [answered]
Hey,

there seem to be quite a few options to skip steps on the optimize process:
--simplify false --prune false --weld false --join false --flatten false , etc...

however --resample false doesn't seem to be an accepted parameter. Is there an alternative that would work?

Thank you!

**Comment by donmccurdy** [comment]
Hi @ketz! Not currently, but if you'd like to open a PR similar to https://github.com/donmccurdy/glTF-Transform/pull/1551 that would be welcome.

**Reply by ketz** [reply, marked as answer]
Done! https://github.com/donmccurdy/glTF-Transform/pull/1587
